src/openforms/js/components/admin/form_design/registrations/objectsapi/ObjectsApiOptionsFormFields.js

```diff
@@ -47,6 +47,7 @@ const ObjectsApiOptionsFormFields = ({index, name, schema, formData, onChange})
         } else {
           delete draft.variablesMapping;
         }
+        delete draft.productaanvraagType;
```

Comment:
Maybe nice to add a check in the story that this field is not present?

Reply:
Those kind of checks are a bit fragile, you can easily satisfy them by changing a translation to something slightly different. I trust the review process more on this :)